UMN-AutoPackager/public/Get-RedirectedUri.ps1

```diff
@@ -0,0 +1,48 @@
+function Get-RedirectedUri {
```

Comment:
This function needs to be moved to the /private directory.

Reply:
Will be moved on next update. Resolving conversation.